contracts/token/non-fungible/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+//! # Non-Fungible Token Contract Module.
```

Comment:
To be adapted for Non-Fungible, removing refs to fungible tokens and removing the mention of SEP-39, because this is not compatible with it.

Reply:
addressed in [7552cf7](https://github.com/OpenZeppelin/stellar-contracts/pull/75/commits/7552cf74f0cf98b1838cc4af1e01e3451fd69904)